repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/AbstractRepairStep.java

```diff
@@ -174,13 +176,11 @@ protected void createPullRequest(List<File> patchList, int nbPatch) throws IOExc
                 GHRepository originalRepository = github.getRepository(this.getInspector().getRepoSlug());
                 GHRepository ghForkedRepo = originalRepository.fork();
 
-                String base = this.getInspector().getBuggyBuild().getBranch().getName();
+                String base = this.getInspector().getBuggyBuild() == null ? ((JenkinsProjectInspector)this.getInspector()).getCheckoutBranchName() : this.getInspector().getBuggyBuild().getBranch().getName();
                 String head = ghForkedRepo.getOwnerName() + ":" + branchName;
-
-                String travisURL = Utils.getTravisUrl(this.getInspector().getBuggyBuild().getId(), this.getInspector().getRepoSlug());
-
-                String prText = String.format(TEXT_PR, travisURL, this.getInspector().getRepoSlug(), this.getInspector().getRepoSlug());
-
+                String travisURL = this.getInspector().getBuggyBuild() == null ? "" : Utils.getTravisUrl(this.getInspector().getBuggyBuild().getId(), this.getInspector().getRepoSlug());
+                String baseString = this.getInspector().getBuggyBuild() == null ? "This PR has been created automatically by [repairnator](https://github.com/eclipse/repairnator).\n" : TEXT_PR;
+                String prText = String.format(baseString, travisURL, this.getInspector().getRepoSlug(), this.getInspector().getRepoSlug());
                 GHPullRequest pullRequest = originalRepository.createPullRequest("Patch proposal", head, base, prText);
```

Comment:
PR Header should be "Patch found by Repairnator with repair tool"+toolName

PR description should contain the name+link of the failing PR/Build, with the failing test names that are fixed.